processing/src/main/java/io/druid/query/aggregation/post/DoubleGreatestPostAggregator.java

```diff
@@ -37,14 +38,7 @@
 
 public class DoubleGreatestPostAggregator implements PostAggregator
 {
-  private static final Comparator COMPARATOR = new Comparator()
-  {
-    @Override
-    public int compare(Object o, Object o1)
-    {
-      return ((Double) o).compareTo((Double) o1);
-    }
-  };
+  private static final Comparator<Number> COMPARATOR = Comparator.nullsFirst(Comparator.comparingDouble(Number::doubleValue));
```

Comment:
Line longer than 120 cols. Might also be in other similar classes, I didn't check.

Reply:
done.